packages/pigeon/lib/ast.dart

```diff
@@ -86,9 +91,12 @@ class Api extends Node {
   /// The name of the Dart test interface to generate to help with testing.
   String? dartHostTestHandler;
 
+  /// List of documentation comments, seperated by line.
+  List<String> documentationComments;
```

Comment:
I think it's worth fleshing this out with discussion of the non-obvious whitespace thing. E.g.:

```
/// List of documentation comments, separated by line.
///
/// Lines should not include the comment marker itself, but should include any
/// leading whitespace, so that any indentation in the original comment is preserved.
/// For example: [" List of documentation comments, separated by line.", ...]
```
```